src/transaction/log_impl.h

```diff
@@ -950,7 +944,7 @@ extern void logtb_set_to_system_tran_index (THREAD_ENTRY * thread_p);
 extern LOG_LSA *logtb_find_largest_lsa (THREAD_ENTRY * thread_p);
 #endif
 extern int logtb_set_num_loose_end_trans (THREAD_ENTRY * thread_p);
-extern void log_find_unilaterally_largest_undo_lsa (THREAD_ENTRY * thread_p, LOG_LSA & max_undo_lsa);
+extern void log_find_unilaterally_largest_undo_lsa (THREAD_ENTRY * thread_p, LOG_LSA & max_undo_lsa); // TODO: move header and impl to log_recovery
```

Comment:
this one is implemented in `log_tran_table` but only used in `log_recovery`

Reply:
The implementation too should be moved to log_recovery.c.

Reply:
Done